Multisig/UI/UI Library/RefreshableScrollView.swift

```diff
@@ -16,7 +16,12 @@ struct RefreshableScrollView<Content: View>: View {
     @State private var frozen: Bool = false
     @State private var rotation: Angle = .degrees(0)
 
-    var threshold: CGFloat = 80
+    var threshold: CGFloat = 70
+
+    // We will begin rotation, only after we have passed
+    // 60% of the way of reaching the threshold.
+    var thresholdRubicon = 0.6
```

Comment:
Rubicon? Really? 😄 sorry, this is also not so explanatory, please make it boring 😄